<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>365</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/408.bmp</normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/408.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>QList</string>
      </attribute>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>初始化列表</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>清除列表</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>插入项</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>删除项</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>添加项</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>365</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/408.bmp</normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/408.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>Page1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>365</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/408.bmp</normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/408.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>Page2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>QList_1</string>
      </attribute>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>371</width>
         <height>291</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>New Item 1</string>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/724.bmp</normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/724.bmp</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>New Item 2</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/724.bmp</normaloff>../../../彩图和配套源代码/chap03Widgets/Demo3_8ListWidget/QtApp/images/724.bmp</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>341</width>
         <height>141</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>当前项变化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>可编辑</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="toolButton_6">
             <property name="text">
              <string>SelectItem</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_7">
             <property name="text">
              <string>SelectAll</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/images/412.bmp</normaloff>:/icons/images/412.bmp</iconset>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_8">
             <property name="text">
              <string>SelectNone</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>QList_2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>QList_3</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_InitList"/>
   <addaction name="action_ClearList"/>
   <addaction name="action_Insert"/>
   <addaction name="action_Add"/>
   <addaction name="action_DeleteItem"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_InitList">
   <property name="icon">
    <iconset theme="删除">
     <normaloff>images/101.bmp</normaloff>images/101.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_ClearList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/images/delete1.bmp</normaloff>:/icon/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
  </action>
  <action name="action_Insert">
   <property name="icon">
    <iconset>
     <normaloff>images/306.bmp</normaloff>images/306.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="icon">
    <iconset>
     <normaloff>images/316.bmp</normaloff>images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
  </action>
  <action name="action_DeleteItem">
   <property name="icon">
    <iconset>
     <normaloff>images/318.bmp</normaloff>images/318.bmp</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="action_All">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="action_None">
   <property name="text">
    <string>全不选</string>
   </property>
  </action>
  <action name="action_Invs">
   <property name="text">
    <string>反选</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
